src/index.js

```diff
@@ -37,19 +38,35 @@ function getAlgoliaSearch(params) {
   return algoliasearch(apiKey, appID);
 }
 
-function getAffectedPaths(params /* , config */) {
-  if (params.mountpoint) {
-    // TODO: translate external paths and check domain in config
-  }
-  const {
-    path: singlePath,
-    paths: multiplePaths,
-  } = params;
+/**
+ * Return source hash for an item.
+ */
+function getSourceHash(driveId, itemId) {
```

Comment:
what about other sources? eg. github paths, google doc ids ?

Reply:
I'm not aware that those are already planned to invoke the `owner-repo-ref` trigger that eventually invokes this action - do we have a format definition for those?

Reply:
that's the idea of using the payload described [here](https://github.com/adobe/helix-home/blob/master/docs/observation/payload.md) ... what do you suggest we should do ?

but I guess, the source-hash is specific to the onedrive, used to delete the items, right?

Reply:
so maybe I was wrong...and we should include the `sourceHash` in the change...maybe as _additional properties_ that each provider can include in the index.